### PR TITLE
NEST-318: Adding a theme for media theme development

### PR DESCRIPTION
**Pull request by barthamark**
NEST-318

### CODE REVIEW

File: src/DotNest.Core.SDK.Web/Program.cs [hunk 1]
```diff
@@ -13,12 +13,8 @@
...
         .AllowMiniProfilerOnAdmin()
         .AddDatabaseShellsConfigurationIfAvailable(configuration)
-        .ConfigureSmtpSettings(overrideAdminSettings: false);
-
-    if (builder.Environment.IsDevelopment())
-    {
-        orchardCoreBuilder.AddSetupFeatures("OrchardCore.AutoSetup");
-    }
+        .ConfigureSmtpSettings(overrideAdminSettings: false)
+        .AddSetupFeatures("OrchardCore.AutoSetup");
```
Comment (wAsnk) [inline src/DotNest.Core.SDK.Web/Program.cs]: Why did you delete the condition for autosetup?

File: src/Themes/DotNest.Core.SDK.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json [hunk 2]
```diff
@@ -127,29 +125,49 @@
...
+          "executionid": "DotNest.Core.SDK.Web",
+          "name": "Lombiq.BaseTheme.LayersAndZones"
+        }
+      ]
+    },
+    {
+      "name": "media",
+      "Files": [
+        {
+          "SourcePath": "_MediaTheme/wwwroot/example.png",
```
Comment (wAsnk) [inline src/Themes/DotNest.Core.SDK.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json]: There are no such files on the source path. The setup fails.

**Review by wAsnk — CHANGES_REQUESTED** [review]

File: src/DotNest.Core.SDK.Web/Program.cs (hunk 1 above)
Comment (barthamark) [reply src/DotNest.Core.SDK.Web/Program.cs]: I'll revert it. I was doing some highly qualified trial and error to fix something.

File: src/Themes/DotNest.Core.SDK.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json (hunk 2 above)
Comment (barthamark) [reply src/Themes/DotNest.Core.SDK.Theme/Recipes/DotNest.Core.SDK.Setup.recipe.json]: Yeah I'll change it to Assets instead of wwwroot. It was kind of convenient to go with folder names that we are "used to" but wwwroot is actually ignored in most cases so it makes sense to go with other folder names.

**Review by wAsnk — APPROVED** [review]
